### DISCUSSION TITLE
刚刚过完4.2剧情，人傻了

### DISCUSSION THREAD

**Discussion by Klein124284420** [share121/inter-knot#3008] [其他]
![6f046524de0826a455b1a6a2bfda3bc](https://github.com/user-attachments/assets/ecef1789-7da0-473e-ae9f-0f8077cc4333)
<img src="https://bbs-static.miyoushe.com/static/2023/11/17/e0564870de2f3d1c39dfc41a8dcd9d2b_383241188874318913.png" alt="芙宁娜-好困难" width="75" >

**Comment by Klein124284420** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/07/19/76665409ea9ee0bdfd967268846ed366_4326264775755692489.png" alt="希儿_委屈" width="75" >

**Comment by RaphaelHider** [comment]
水龙水龙别哭啦

**Reply by Nie7bai** [reply]
朋友，你要杀了我吗？

**Reply by 1492830678** [reply]
马卡龙别哭啦<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by weimaojun** [comment]
n

**Comment by warning-Kirov-reporting** [comment]
好些天没缓过来<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by Kamisato100** [comment]
水元素充盈了罢<img src="https://bbs-static.miyoushe.com/static/2023/08/23/b936459d0514da31651d086a2c507345_3938242848160050564.png" alt="芙宁娜-乐" width="75" >
